roles/confluent.ksql/defaults/main.yml

```diff
@@ -6,12 +6,19 @@ ksql:
   systemd_override: /etc/systemd/system/confluent-ksql.service.d
   user: cp-ksql
   group: confluent
+  keytab: /etc/security/keytabs
```

Comment:
should be shared var

Reply:
please file a feature request for this, as this wasn't designed to be configurable in this version of the playbooks. We can certainly do this in a future release.